src/olympia/amo/tests/__init__.py

```diff
@@ -852,7 +852,8 @@ def refresh(cls, index='default', timesleep=0):
     @classmethod
     def reindex(cls, model, index='default'):
         # Emit post-save signal so all of the objects get reindexed.
-        [o.save() for o in model.objects.all()]
+        manager = getattr(model, 'unfiltered', getattr(model, 'objects'))
```

Comment:
I think you could easily do `getattr(model, 'unfiltered', model.objects)` without there being any side-effects imho.